code/modules/mob/living/carbon/human/species/unathi.dm

```diff
@@ -60,46 +60,40 @@
 
 /datum/species/unathi/on_species_gain(mob/living/carbon/human/H)
 	..()
-	var/datum/action/innate/tail_lash/lash = new()
-	lash.Grant(H)
+	var/datum/action/innate/ignite/fire = new()
+	fire.Grant(H)
 
 /datum/species/unathi/on_species_loss(mob/living/carbon/human/H)
 	..()
-	for(var/datum/action/innate/tail_lash/lash in H.actions)
-		lash.Remove(H)
-
-/datum/action/innate/tail_lash
-	name = "Tail lash"
-	icon_icon = 'icons/effects/effects.dmi'
-	button_icon_state = "tail"
+	for(var/datum/action/innate/ignite/fire in H.actions)
+		fire.Remove(H)
+
+/datum/action/innate/ignite
+	name = "Ignite"
+	desc = "Requires you to drink welding fuel beforehand"
+	icon_icon = 'icons/obj/cigarettes.dmi'
+	var/cooldown = 0
```

Comment:
The `cooldown` var seems unused, presumably a remnant from when you wanted to make it cooldown-based?